packages/pricing/integration-tests/__tests__/services/currency/index.spec.ts

```diff
@@ -183,7 +183,7 @@ describe("Currency Service", () => {
         error = e
       }
 
-      expect(error.message).toEqual('"currencyCode" must be defined')
+      expect(error.message).toEqual("currency - code must be defined")
```

Comment:
question: How are these error messages generated?

Reply:
You can find them in the internal service factory. But before they where trying to provide the name of the expected argument except that it doesn't work anymore, so now it shows the primary key(s) missing